aiida/restapi/common/utils.py

```diff
@@ -845,3 +847,17 @@ def list_routes():
         output.append(line)
 
     return sorted(set(output))
+
+
+@decorator
+def close_session(wrapped, _, args, kwargs):
```

Comment:
I guess you are dead-set on keeping this here as a utility then? :)

Reply:
I don't see the reason to make it a class static method in any case. However, I would _really_ like to move it some place else, if you have a good argument :)